web_src/js/components/ActivityHeatmap.vue

```diff
@@ -37,8 +38,14 @@ export default {
     ],
     endDate: new Date(),
     locale: {
-      contributions: 'contributions',
-      no_contributions: 'No contributions',
+      months: new Array(12).fill().map((_, idx) => formatMonth(idx)),
+      days: new Array(7).fill().map((_, idx) => formatDay(idx)),
+      contributions: window.config.i18n.contributions.toLocaleLowerCase(getCurrentLocale()),
+      no_contributions: window.config.i18n.no_contributions,
+      contributions_in_the_last_12_months: window.config.i18n.contributions_in_the_last_12_months.toLocaleLowerCase(getCurrentLocale()),
+      on: window.config.i18n.on.toLocaleLowerCase(getCurrentLocale()),
+      less: window.config.i18n.less,
+      more: window.config.i18n.more,
```

Comment:
You can destructure these at the start of the function to make it more readable:

```js
const {i18n} = window.config;